src/nn/split_into.rs

```diff
@@ -17,7 +17,7 @@ use crate::prelude::*;
 /// let _: (Tensor1D<3>, Tensor1D<7>) = model.forward(Tensor1D::<5>::zeros());
 /// ```
 #[derive(Debug, Default, Clone)]
-pub struct SplitInto<T>(T);
+pub struct SplitInto<T>(pub(crate) T);
```

Comment:
we can probably just make this `pub`. I think all the nn modules can just pub all their underlying modules.